### PR TITLE
New patch release of Golang: v1.21.11

### PR DESCRIPTION
**Pull request by sp1999**
Update EKS Go Patch Version: v1.21.11-12
SPEC FILE STILL NEEDS THE '%changelog' UPDATED
PLEASE UPDATE WITH THE FOLLOWING FORMAT
```
* Wed Sep 06 2023 Cameron Rozean <rcrozean@amazon.com> - 1.20.8-1
- Bump tracking patch version to 1.20.8 from 1.20.7
```

### CODE REVIEW

File: projects/golang/go/1.21/rpmbuild/SPECS/golang.spec [hunk 1]
```diff
@@ -527,7 +527,10 @@ fi
 %endif
 
 %changelog
-* Fri May 10 2024 Sajia Zafreen <szafreen@amazon.com> - 1.21.10-10
+* Fri Jun 05 2024 Saurabh Parekh <sjparekh@amazon.com> - 1.21.11-12
```
Comment (rcrozean) [inline projects/golang/go/1.21/rpmbuild/SPECS/golang.spec:530]: This is causing a build warning: `warning: bogus date in %changelog: Fri Jun 05 2024 Saurabh Parekh <sjparekh@amazon.com> - 1.21.11-12` Can we fix this to be `Wed Jun 05 2024...`

Comment (sp1999) [reply projects/golang/go/1.21/rpmbuild/SPECS/golang.spec:530]: Fixed it in a new PR

**Pull request merged**